contracts/access/Roles.sol

```diff
@@ -31,6 +32,7 @@ library Roles {
      */
     function has(Role storage role, address account) internal view returns (bool) {
         require(account != address(0), "Roles: account is the zero address");
+        // require(role.exists, "Roles: role does not exists");git
```

Comment:
The fact that this check is not really necessary makes me think that perhaps having `.exists` is not the best way to go: it is possible to misuse the `Roles` struct by forgetting to set this field.